src/runtime/client-event-manager.js

```diff
@@ -29,6 +29,28 @@ function createEventErrorMessage(valueName, value) {
   return 'Event has an invalid ' + valueName + '(' + value + ')';
 }
 
+/**
+ * Utility function to create a ClientEvent proto.
+ * @param {!AnalyticsEvent} eventType
+ * @param {!EventOriginator} eventOriginator
+ * @param {?boolean} isFromUserAction
+ * @param {?Object} additionalParameters
+ * @returns {!../api/client-event-manager-api.ClientEvent}
+ */
+export function CreateClientEvent(
```

Comment:
Since this function is for internal use, I would suggest one of the following:

1) Make eventOriginator the last parameter and make it optional.  Default to SWG_CLIENT
2) Create a second createSwgEvent where originator is not passed in and have it call createClientEvent with SWG_CLIENT.  I would expect almost nothing to use this without it being SWG_CLIENT excluding a few lines of code in the logger class.

also, do our utility functions generally start with a lower case or capital letter?  I feel like generally it is pascalCase but maybe I'm wrong.

Reply:
I'd also make isFromUserAction and additionalParameters optional.  The syntax for this is:

{boolean?} which means it can be undefined / not even passed in.

Reply:
Went with making eventOriginator, isFromUserAction, and additionalParameters all optional by marking them as having default values.

Reply:
I think originator should be last because it is the least likely to be used.  isFromUserAction I would actually like to see everything use so it should be second.  additionalParameters will probably not be used internally (much) so it should be third.

Reply:
I like putting isFromUserAction second but left eventOriginator as third position because I don't like the idea of putting an additional param before an enum.

Reply:
Sounds fine to me